.github/workflows/dependencies/Get-MSBuildResults.ps1

```diff
@@ -96,6 +96,24 @@ if (($RangeStart -ne 0) -and ($RangeEnd -ne 0)){
     $workingSet = $output[$RangeStart..$RangeEnd]
 }
 
+# Log working set items prior to filtering
+$workingSet | Write-Host
+
+# Remove duplicated projects
+$projects = @()
+$workingSetTemp = @()
+
+foreach ($item in $workingSet) {
+    if ($projects.Contains($item[2])) {
+        continue
+    }
+    $projects += $item[2]
+    $workingSetTemp += $item
+}
+
+$workingSet = $workingSetTemp

```

Comment:
Doesn't the LocateProjects tool already create the set of unique projects to build?

Reply:
No, it returns each file + project combo